openvpn/middlewares/client/bytescount/middleware.go

```diff
@@ -49,19 +55,21 @@ func (middleware *middleware) Start(commandWriter management.CommandWriter) erro
 }
 
 func (middleware *middleware) Stop(commandWriter management.CommandWriter) error {
+	defer middleware.postStopHandle()
```

Comment:
I imagined this feature as separate middleware, which listens for Openvpn events and hooks on DISCONNECT event or smtg.
Do we need stats about "stoping middleware" or "connection end"?

Reply:
Exactly. But I and @soffokl already discussed it, and decided that he will take a different approach